etc/docker-scripts/bootstrapper.py

```diff
@@ -116,12 +127,9 @@ def get_file_from_object_storage(client, bucket_name, file_to_get):
     print('Get file {} from bucket {}'.format(file_to_get, bucket_name))
     object_to_get = get_object_storage_filename(file_to_get)
 
-    try:
```

Comment:
Why are we removing try/except?

Reply:
This is probably due to a comment I added that has since disappeared.  (why does that happen!?)

The try/catch was removed under the assumption (I asked) whether a non-existent input could be treated as fatal.  Previously, the exception was printed and ignored.   However, if treated as a failure, which it should be IMO, the container will stop and the exception will still be logged by virtue of it being 'unhandled'.

The driving force behind this is that by not having to reference the minion error, both 'get' and 'put' methods wouldn't need to use that stupid deferred `import minio` statement (needed for unit testing) and it didn't seem the handling of that error was adding any value anyway - assuming that `get` should be treated as seriously as `put` - which merely re-raised the caught exception.

Reply:
these should be treated as fatal in my opinion as well. a missing file/key should not be allowed to continue